modules/ROOT/pages/basketball-robot-guide/mechanics/cam.adoc

```diff
@@ -27,24 +29,62 @@ to make a model that you can edit without affecting the original model.
 
 image::fusion_360_cam_manufacturing_model.png[]
 
-If you have a multipart assembly you can use the "Arrange" functionality from Fusion 360 preview features
-to automatically arrange and flatten the assembly.
+Parts that will not be milled out should be removed from
+the manufacturing model.
 
-A video tutorial on how to enable and use the "Arrange" functionality can be found here: +
+The manufacturing model can be edited by right-clicking on it and selecting
+"Edit Manufacturing Model".
+
+image::fusion_360_cam_manufacturing_model_edit.png[]
+
+== Arrange
+
+Using the Arrange feature, flat parts can be placed down and set up for milling.
+
+Arranging the parts will be done in the Manufacturing Model.
+
+image::fusion_360_cam_manufacturing_model_edit.png[]
+
+Using Arrange in Manufacturing Model is more convenient,
+and does not break operations when changing the arrangement of parts.
+
+A video tutorial on how to enable and use the "Arrange" functionality
+can be found here: +
 https://youtu.be/7egLufCg5tk
 
+You should also check out these two guides below
+
+Follow the *Arrange in the Manufacture workspace* part +
+https://help.autodesk.com/view/fusion360/ENU/?guid=NST-ARR-MFG-MDL[Fusion Help | Arrange in the Design or Manufacture workspace]
+
+And this guide +
+https://help.autodesk.com/view/fusion360/ENU/?guid=NST-ARR-SELECT[Fusion Help | Arrange components on a flat surface]
+
+
+The settings below can be used. These settings can be changed later,
+i.e the length and width, depending on the dimensions of
+the actual available material.
+Make sure that the "Allow Partial Arrange" is unticked, this will let Fusion
+raise an error message if the selected parts do not fit into the area.
+"Auto Preview" can also be unticked so Fusion doesn’t calculate a new arrangement
+when selecting a new part or changing the settings.
+
+image::fusion_360_cam_arrange_settings.png[]
+
 == Job setup
 
 First thing to do is to create a new setup for your job.
 For this, choose SETUP -> New Setup from toolbar.
 
 image::fusion_360_cam_new_setup.png[]
 
-You should now set stock properties and origin point (the point where the milling starts).
-Usually the starting (work zero) point should be in the bottom left corner of your part on top of the stock.
-Z axis should point up, X and Y axis should point along the sides of your part.
-Correct orientation can be chosen by selecting relevant properties under Work Coordinate System (WCS).
-Depending on how the design is oriented, you might need to select different options.
+You should now set stock properties and origin point (the point where the
+milling starts). Usually the starting (work zero) point should be in the bottom
```

Comment:
Line lengths were mostly ok before.

LIne length of 120 symbols (default in JetBrains IDEs) is fine.

Each sentence should start on a new line.